tests/manual/memory/memory.md

```diff
@@ -0,0 +1,31 @@
+1. Open DevTools on Chrome.
+2. Go to Memory tab.
+3. Select "Heap snapshot" profiling type.
+4. Click "Collect Garbage" (trash icon) and "Clear all profiles" (don't go icon).
+5. Record heap snapshot ("Take heap snapshot" a record icon).
+6. Repeat multiple times:
+    - click "Init editor",
+    - wait to editor show up ,
+    - click "Destroy editor".
+7. Record heap snapshot again.
+8. Compare Snapshot 1 with Snapshot 2 and look for:
+    - "detached" - HTML Elements/DOM nodes
+    - "EventListener" - there should be only 1
+    - Any CKEditor5 classes instances like "Editor" or "LivePosition"
+
+## Notes:
+
+* Browser extensions might attach event listeners to the DOM so the safest way to run this test is by running Chrome from command line:
+
+    ```
+    google-chrome \
+        --enable-precise-memory-info \
+        --disable-extensions \
+        --disable-plugins \
+        --incognito \
+        http://localhost:8125 
```

Comment:
```suggestion
        http://localhost:8125
```

Extra tailing space. This behavior is defined in our [`.editorconfig`](https://github.com/ckeditor/ckeditor5/blob/5948bcff409d79bb6b1f8b6dce949eb1d5c6d38b/.editorconfig#L11) double check your editor settings.

Reply:
Actually it might be even more convenient to link straight to manual test: `http://localhost:8125/ckeditor5/tests/manual/memory/memory.html`?